manim/utils/color.py

```diff
@@ -437,14 +437,17 @@ def average_color(*colors: Color) -> Color:
     return rgb_to_color(mean_rgb)
 
 
-def random_bright_color() -> Color:
-    color = random_color()
+def random_bright_color(random_seed=None) -> Color:
+    color = random_color(random_seed)
```

Comment:
It is better to be explicit when passing around arguments like that.

```suggestion
    color = random_color(random_seed=random_seed)
```